spec/features/course/group_management_spec.rb

```diff
@@ -16,7 +16,7 @@
         expect(page).to have_link(nil, href: new_course_group_path(course))
 
         groups.each do |group|
-          expect(page).to have_selector('td', text: group.name)
+          expect(page).to have_selector('th', text: group.name)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.